weaver/utils/dataset.py

```diff
@@ -129,8 +129,10 @@ def _preprocess(table, data_config, options):
     return table, indices
 
 
-def _load_next(data_config, filelist, load_range, options):
+def _load_next(data_config, filelist, split_num_info, load_range, options):
     load_branches = data_config.train_load_branches if options['training'] else data_config.test_load_branches
+    split_num, split_tot_num = split_num_info
+    filelist = filelist[split_num::split_tot_num]
```

Comment:
```suggestion
    i_split, split_num = split_num_info
    filelist = filelist[i_split::split_num]
```

to match the naming below